resources/shuttle-shared-db.mdx

```diff
@@ -20,6 +20,8 @@ Each type of shareable database is behind its own feature flag and macro attribu
 | Postgres | `postgres`   | `shuttle_shared_db::Postgres` |
 | MongoDB  | `mongodb`    | `shuttle_shared_db::MongoDb`  |
 
+<Warning>We are planning to deprecate the MongoDB resource soon, with upcoming features to replace it.</Warning>
```

Comment:
Thanks! Perhaps something like this would be good, so we can get some data on what they need?
```suggestion
<Warning>The MongoDB resource will be deprecated in the near future. If this resource is important to you, please open a [GitHub issue](https://github.com/shuttle-hq/shuttle/issues) detailing your use-case.</Warning>
```

Reply:
Makes sense to me - have committed